turf/boolean_disjoint/_boolean_disjoint.py

```diff
@@ -0,0 +1,198 @@
+from typing import Any, List, Sequence, Union
+
+from turf.helpers import (
+    Feature,
+    LineString,
+    MultiLineString,
+    MultiPolygon,
+    Point,
+    Polygon,
+)
+
+from turf.helpers import line_string, point, polygon
+
+from turf.boolean_point_in_polygon import boolean_point_in_polygon
+from turf.boolean_point_on_line import boolean_point_on_line
+from turf.invariant import get_coords_from_features, get_geometry_type
+from turf.polygon_to_line import polygon_to_line
+from turf.line_intersect import line_intersect
+
+from turf.utils.error_codes import error_code_messages
+from turf.utils.exceptions import InvalidInput
+
+
+def boolean_disjoint(feature_1: Any, feature_2: Any) -> bool:
+    """
+    Returns true if the intersection of the two geometries is an empty set.
+
+    :param feature_1: {GeoJSON} feature_1 any Feature or Geometry
+    :param feature_2: {GeoJSON} feature_2 any Feature or Geometry
+    :return: boolean True/False if features are disjoint
+    """
+    flat_feature_1 = flatten_feature(feature_1)
+    flat_feature_2 = flatten_feature(feature_2)
+
+    for flat_1 in flat_feature_1:
```

Comment:
The variable `is_disjoint` should be initialized:

```python
is_disjoint = True
```

Reply:
it is initialized in the subfunction, but I can also add the variable here.